documentation/asciidoc/computers/raspberry-pi/power-supplies.adoc

```diff
@@ -70,7 +75,7 @@ Exactly how much current (mA) the Raspberry Pi requires is dependent on what you
 
 Raspberry Pi have developed their own power supplies for use with all models. These are reliable, use heavy gauge wires and are reasonably priced.
 
-For Raspberry Pi 0-3, we recommend our https://www.raspberrypi.com/products/micro-usb-power-supply/[2.5A micro USB Supply]. For Raspberry Pi 4, we recommend our https://www.raspberrypi.com/products/type-c-power-supply/[3A USB-C Supply]
+For Raspberry Pi 0-3, we recommend our https://www.raspberrypi.com/products/micro-usb-power-supply/[2.5A micro USB Supply]. For Raspberry Pi 4, Pi 400, we recommend our https://www.raspberrypi.com/products/type-c-power-supply/[3A USB-C Supply]
```

Comment:
As earlier, "and" instead of "," please. And there ought to be a full-stop at the very end. :slightly_smiling_face: